AstarteDeviceSDKCSharp/Transport/MQTT/AstarteMqttV1Transport.cs

```diff
@@ -26,6 +26,7 @@
 using MQTTnet;
 using MQTTnet.Extensions.ManagedClient;
 using MQTTnet.Protocol;
+using System.Diagnostics;
```

Comment:
Is it right to add `System.Diagnostics` deps? Or  is it only for debugging purpose?

Reply:
We need this namespace for [Trace.WriteLine](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.trace.writeline?view=net-6.0) method.